package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "rapidpro2pg",
+  "version": "0.0.1",
+  "description": "Adapter that pulls data from RapidPro and makes it available in PostgreSQL for ease of analysis.",
+  "main": "src/index.js",
+  "dependencies": {
+    "loglevel": "^1.7.1",
+    "node-fetch": "^2.6.1",
+    "pg": "^8.6.0",
+    "pg-format": "^1.0.4",
+    "postgrator": "^4.1.1"
+  },
```

Comment:
One more think, due that is the first stable version we are going to release, can we fix the dependencies with up to date versions? I got this message installing the dependencies: `8 vulnerabilities (3 low, 1 moderate, 4 high)`, and maybe just running `npm audit fix` fix it.

Reply:
Done! All vulnerabilities were caused by `express`. And installing `4.17` instead of `4.14` fixed it. 
Interesting that I had used `npm install express` to get the vulnerable version.